### DISCUSSION TITLE
Kiwi Browser Support

### DISCUSSION THREAD

**Discussion by maxelo** [darkreader/darkreader#6911] [Ideas]
Long time Dark Reader user here.
I've been using it on Chrome and Firefox version for a long time. Exporting and Importing my rules to my android phones.
Then along came Kiwi Browser (94.0.460656). Blazing fast browser with webkit and chrome web store extension support like ublock origin and tampermonkey =)
I know that there is also a dark mode support on kiwi browser. But I doesn´t get close to the customization of Kiwi Browser.
Using Dark Reader today on Kiwi Browser only lets you push one toggle before the extension gets stuck (like all or nothing gets darked). 
I would love to know if there any suport planning for kiwi browser? Maybe a way to just import a .json configuration

thanks for your work!

**Comment by Gusted** [comment]
HI!

Can you turn this into a bug report, as we do _officially_ support Kiwi Browser.
I can reproduce the bug about not being able to change any setting.

Regards,
Gusted

**Reply by maxelo** [reply]
ok, I'll do it 
thank you =)

**Reply by maxelo** [reply]
Just did a bug report
https://github.com/darkreader/darkreader/issues/6913
